### ISSUE TITLE
Add "HTML Anchor" and "Additional CSS class(es)" to List View kebab menu

### ISSUE THREAD

**Issue by jdcohan** [WordPress/gutenberg#41291]
## What problem does this address?
Adding HTML anchors and Additional CSS class(es) is **something I do all the time**. I often add these values **after** I have added blocks. I find it frustrating and inefficient to have to select a block, then navigate down to the bottom of the settings panel, then expand the "Advanced" section, then add the anchor and class(es) — and then do it all again for each block.

## What is your proposed solution?
Adding "HTML Anchor" and "Additional CSS class(es)" to the list view kebab menu would make things so much easier.

**Comment by deborah86** [comment]
I am not sure if adding additional menu options is the best solution to your problem.

For CSS classes you are able to create reusable blocks and use the same blocks with those classes again. I believe they will be phasing this option out soon when they make the final transition to FSE.

A problem is the editor removed the ability to collapse some settings. It also doesn't remember what settings were expanded or collapsed. If this were fixed, the editing experience would be greatly improved.

**Comment by jdcohan** [comment]
> For CSS classes you are able to create reusable blocks and use the same blocks with those classes again.

A. I'm not interested in employing reusable blocks for this purpose.

B. I'm not adding the SAME classes or IDs to multiple blocks. I'm not looking for the equivalent of a "Bulk Edit" process. I'm looking for a more efficient workflow for adding those "Advanced" values to blocks.

**Comment by SergeGusev** [comment]
Good idea, I waste time for these unnessesary steps too.

**Comment by deborah86** [comment]
@jdcohan They are phasing out the old editor for the FSE. Adding CSS classes on the blocks will probably be phased out because it is a feature of the old editor.

The best solution would be for them to allow you to collapse/expand all options in the Block Settings menu and then save your settings. That way you can collapse all the other settings you don't want other than the advanced settings. Adding the same options to multiple menus isn't really a solution to your problem.

**Comment by jdcohan** [comment]
@deborah86 
> **Adding CSS classes on the blocks will probably be phased out** because it is a feature of the old editor.

Where do you get that information? You're saying Gutenberg won't allow us to add CSS classes to blocks? That is both hard to believe and (IMHO) preposterous. But I'm not saying you're wrong. I'm just hoping you are.

> The best solution would be for them to allow you to collapse/expand all options in the 
> Block Settings menu and then save your settings. 

That would work for me. But if you're saying **Adding CSS classes on the blocks will probably be phased out**, then there is no solution for me.

I'm confused.

**Comment by deborah86** [comment]
@jdcohan The new editor does not have the ability to add custom CSS like the customizer. I did research and it is possible in the Theme File Editor. They are phasing out the Customizer. So, yes, it is still possible to add custom CSS but you will need to use the Theme File Editor after they phase out the Customizer.

**Comment by jdcohan** [comment]
@deborah86 

(Emphasis added by me.)

> @jdcohan The new editor does not have the ability to add **custom CSS like the customizer**.

Ok. I think I see the confusion.

You are now saying we won't be able to add **custom CSS**. I'm not talking about adding custom CSS *rules*. I'm talking about adding **Additional CSS Class(es)** — class **names**.

**Comment by iwasthesword** [comment]
This would indeed be great, as someone who uses a lot of classes for templating (bootstrap) it would save a LOT of time and make workflow way easier. Maybe it could be an optional settings.

**Comment by jdcohan** [comment]
@iwasthesword 

> Maybe it could be an optional settings.

Good idea. In fact, that makes me think of another possible enhancement: allowing users to show/hide **any** List View kebab menu item from the Preferences panel. Right? If you want to propose it, go for it! Otherwise, I might...
